<launch>

    <arg name="markerSize"      default="0.027"/>    <!-- in m -->
    <arg name="side"             default="left"/>
    <arg name="ref_frame"       default=""/>  <!-- leave empty and the pose will be published wrt param parent_name -->


    <node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher">
        <remap from="/camera_info" to="camera/color/camera_info"/>
        <remap from="/image" to="/camera/color/image_raw" />
        <param name="image_is_rectified" value="False"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="robot1/camera_link"/>
    </node>

</launch>